components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -84,19 +84,37 @@ export class GithubApp {
         });
 
         app.on('installation.created', async ctx => {
-            const accountId: string = `${ctx.payload.installation.account.id}`;
+            const targetAccountName = `${ctx.payload.installation.account.login}`;
             const installationId = `${ctx.payload.installation.id}`;
-            const senderId = `${ctx.payload.sender.id}`;
-            const user = await this.userDB.findUserByIdentity({ authProviderId: this.env.githubAppAuthProviderId, authId: accountId });
```

Comment:
`accountId` is the target account id, e.g. from `github.com/gitpod-io`. this are not valid coordinates for user accounts.

Reply:
See the documentation URL at L.90.